runtime/agung/Cargo.toml

```diff
@@ -214,10 +214,16 @@ version = '0.0.2'
 
 [dependencies.peaq-pallet-storage]
 git = 'https://github.com/peaqnetwork/peaq-storage-pallet.git'
```

Comment:
After we merge the MR in the peaq-storage-pallet, please change the branch here because we'll delete the feature branch.